### PR TITLE
Life Science Login provider

### PR DESCRIPTION
**Pull request by mzur**
This adds a new provider for [Life Science Login](https://lifescience-ri.eu/ls-login/) (from the European life science infrastructure).

### CODE REVIEW

File: src/LifeScienceLogin/README.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    ],
+];
+```
+
+### Usage
+
+You should now be able to use the provider like you would regularly use Socialite (assuming you have the facade installed):
+
+```php
+return Socialite::driver('lifesciencelogin')->redirect();
```
Comment (atymic) [inline src/LifeScienceLogin/README.md:41]: Add returned fields please

File: src/LifeScienceLogin/Provider.php [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+namespace SocialiteProviders\LifeScienceLogin;
+
+use Exception;
+use Illuminate\Support\Carbon;
+use Illuminate\Support\Facades\Cache;
+use Laravel\Socialite\Two\InvalidStateException;
+use SocialiteProviders\Manager\OAuth2\AbstractProvider;
+use SocialiteProviders\Manager\OAuth2\User;
+
+class Provider extends AbstractProvider
```
Comment (atymic) [inline src/LifeScienceLogin/Provider.php:12]: Can we get an `@see` for the docs URL

File: src/LifeScienceLogin/composer.json [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+            "email": "martin@cebitec.uni-bielefeld.de"
+        }
+    ],
+    "support": {
+        "issues": "https://github.com/socialiteproviders/providers/issues",
+        "source": "https://github.com/socialiteproviders/providers",
+        "docs": "https://socialiteproviders.com/lifesciencelogin"
+    },
+    "require": {
+        "php": "^7.4 || ^8.0",
```
Comment (atymic) [inline src/LifeScienceLogin/composer.json:25]: Drop 7.x

**Review by atymic — CHANGES_REQUESTED** [review]

**Review by atymic — APPROVED** [review]

**Comment by mzur** [comment]
Thanks! I'll also create a PR for the website. Do I have to do anything for the package to be published to Packagist?

**Comment by atymic** [comment]
Oops, forgot. Done :)

**Pull request merged**